mmv1/products/redis/terraform.yaml

```diff
@@ -61,6 +61,14 @@ overrides: !ruby/object:Overrides::ResourceOverrides
           network_name: "redis-test-network"
         test_vars_overrides:
           network_name: 'BootstrapSharedTestNetwork(t, "redis-mrr")'
+      - !ruby/object:Provider::Terraform::Examples
+        name: "redis_instance_cmek"
+        primary_resource_id: "cache"
```

Comment:
Likely we want to add `skip_test: true` here to skip this test as it creates cmek keys that can't be managed by terraform and then it will break the test on subsequent runs. 

Just built and ran this test locally and the test passed.

Reply:
Done